lib/vault/request.rb

```diff
@@ -37,5 +37,7 @@ def extract_headers!(options = {})
         end
       end
     end
+
+    include EncodePath
```

Comment:
Nit, but can we put this right below `private`? Or is this a new convention to put modules included privately at the bottom?